File: src/main/java/org/jdc/template/domain/main/individualquery/IndividualQueryManager.java
/*
 * IndividualQueryManager.java
 *
 * Generated on: 09/17/2014 03:15:27
 *
 */



package org.jdc.template.domain.main.individualquery;


import javax.inject.Inject;

@javax.inject.Singleton
public class IndividualQueryManager extends IndividualQueryBaseManager {

    @Inject
    public IndividualQueryManager() {
    }

    @Override
    public String getQuery() {
        return IndividualQuery.QUERY;
    }

}

File: src/main/kotlin/org/jdc/template/domain/main/individualquery/IndividualQueryManager.kt
/*
 * IndividualQueryManager.kt
 *
 * Generated on: 02/14/2016 07:02:10
 *
 */



package org.jdc.template.domain.main.individualquery

import org.jdc.template.domain.DatabaseManager


@javax.inject.Singleton
class IndividualQueryManager : IndividualQueryBaseManager {


    @javax.inject.Inject
    constructor(databaseManager: DatabaseManager): super(databaseManager) {
    }

    override fun getQuery(): String {
        return IndividualQuery.QUERY
    }


}